### ISSUE TITLE
LDAP / Active Directory Support

### ISSUE THREAD

**Issue by ciph** [opensupports/opensupports#69]
Would love to see LDAP / Active Directory Support in the future.

**Comment by mvldebian** [comment]
+1 to support this feature in the feature

**Comment by Tacioandrade** [comment]
+1 This is an essential functionality to be able to deploy to any minimally structured network.

**Comment by wesleigh** [comment]
@opensupports - is this feature being considered?

**Comment by ciph** [comment]
@ivandiazwm @7JuanCruz7 @guillegiu @mredigonda - Is this something that we might see in the future?

**Comment by mredigonda** [comment]
It's definitely something that you will see in the future, given the current interest for this feature. 
It will take some time however, since we already have a [pool of enhancements](https://github.com/opensupports/opensupports/projects/1) to make, but we will try to give some priority to this too.

**Comment by carloshilgemberg** [comment]
I have a lot of interest in seeing LDAP in conjunction with OpenSupports. I just have not used the system anymore because of this option. No more Open Supports is fantastic.

**Comment by ciph** [comment]
Thanks so much for taking the time to add this feature. Looking forward to see it working :)

**Comment by igrekkwadrat-admin** [comment]
Same here, really looking forward to this. Any ETA?

**Comment by mengelsen** [comment]
I would love to see LDAP/LDAPS Integration added! The sooner, the better!

**Comment by raspbeguy** [comment]
I really would like to use it but it's impossible for me without LDAP.

**Comment by tuxudo** [comment]
I would really like to see this be a feature as well

**Comment by wokawoka** [comment]
the same, I'm just waiting for the LDAP integration in order to start using opensupports

**Comment by mengelsen** [comment]
Looks like v4.5 was published two days ago! Does this mean that LDAP integration has been incorporated into OpenSupports?!?
 https://github.com/opensupports/opensupports

**Comment by mengelsen** [comment]
> Looks like v4.5 was published two days ago! Does this mean that LDAP integration has been incorporated into OpenSupports?!?
> https://github.com/opensupports/opensupports

Looks like v4.5 does not (yet) offer LDAP user authentication. https://github.com/opensupports/opensupports/releases

**Comment by ciph** [comment]
I gave up hope. More than 2 years ago since I requested this feature.
We started using OTRS a while back.

**Comment by brng1** [comment]
This functionality is essential!

Do you have any plans to implement it?

**Comment by Srpenec** [comment]
Please implement LDAP. It is a very important function.

**Comment by catninelife** [comment]
3 years already, still looking forward the LDAP..

**Comment by LukeD1994** [comment]
Just discovered this, wanted to opt for this solution over others for our internal staff helpdesk. Needs LDAP to get the most out of it. I can hope.

**Comment by compumatter** [comment]
Yep.  Same here.  No LDAP can't shortlist it.

**Comment by fred410** [comment]
Yep, its been just over 5 yrs so far since this was first requested.
We ate looking to replace and migrate off of Jira since they are moving to cloud only; and this product looked promising.
But no LDAP, so it's not likely we will use this. Sad.

**Comment by Geek1405** [comment]
+1
